source/clear-linux/concepts/security.rst

```diff
@@ -16,65 +16,65 @@ within the project's codebase and operating culture.
 Security in Updates
 ===================
 
-The |CL| team believes in the benefits of 
-software security through open sourcing, incremental updates, and 
+The |CL| team believes in the benefits of
+software security through open sourcing, incremental updates, and
 rapidly resolving known security advisories.
- 
+
 
 
 The latest Linux codebase
 -------------------------
 
-|CL| uses the newest version of the Linux kernel which allows the operating 
-system to leverage the latest features from the upstream Linux kernel, 
-including security fixes. 
+|CL| uses the newest version of the Linux kernel which allows the operating
```

Comment:
Please assign * to Linux here.

Reply:
LGTM.

Reply:
Added * to line 25, first instance of Linux* outside of the "Clear Linux*" context. :)